src/SPI.cpp

```diff
@@ -462,6 +462,30 @@ void TSC::SetTouchCoords(u16 x, u16 y)
     NDS.KeyInput &= ~(1 << (16+6));
 }
 
+void TSC::MoveTouchCoords(u16 x, u16 y) // 0 -> negative, 1 -> neutral, 2 -> positive
```

Comment:
Do you think you could make this an `enum` instead of a series of magic numbers? I could easily see some poor bastard trying to pass mouse coordinates in here, wondering why nothing works.

Reply:
Sure. Any preferable place where I should place the enum?

Reply:
`SPI.h` would be my first instinct.

Reply:
What do you think @JesseTG ?

Reply:
That's correct